### ISSUE TITLE
Polaris and BCPL Website - Change Active and Focused Color for Buttons

### ISSUE THREAD

**Issue by sdurphy** [baltimorecounty/BCPL-assets#507]
In Polaris and on the BCPL site, can we change the active and focused color for buttons from the default Bootstrap blue, to dark Turquoise (#137875)? Ex: "Log In Now" button under "Account" in top right of BCPL site.

**Comment by martypowell** [comment]
@sdurphy yes. @jsmith176 if you take this one, this will be applicable for both bcpl and polaris.

**Comment by martypowell** [comment]
@sdurphy we want to use rgb(20,120,118); which is slightly different than the hex you provided above, but already existed.

**Comment by jsmith176** [comment]
@danfox01 This is in both dev and ils-test. Ready for testing.
